Source/FieldSolver/SpectralSolver/SpectralFieldData.H

```diff
@@ -19,7 +19,7 @@ using SpectralField = amrex::FabArray< amrex::BaseFab <Complex> >;
 
 /** Index for the regular fields, when stored in spectral space */
 struct SpectralFieldIndex {
-  enum { Ex=0, Ey, Ez, Bx, By, Bz, Jx, Jy, Jz, rho_old, rho_new, n_fields };
+  enum { Ex=0, Ey, Ez, Bx, By, Bz, Jx, Jy, Jz, rho_old, rho_new, divE, n_fields };
```

Comment:
```suggestion
  enum { Ex=0, Ey, Ez, Bx, By, Bz, Jx, Jy, Jz, rho_old, rho_new, n_fields
              divE=3 }; // divE reuses the memory slot for Bx, since Bx is not used when computing divE
```